implementations/elixir/ockam/ockam/lib/ockam/transport/portal/msgs.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Ockam.Transport.Portal.TunnelProtocol do
```

Comment:
Can we keep the module name consistent with the file name? It's easier to navigate this way.